### DISCUSSION TITLE
Discuss here

### DISCUSSION THREAD

**Discussion by B-R-P** [LangTrans/LangTrans#13] [Off-topic]
Discussions that are not related to LangTrans.

**Comment by B-R-P** [comment]
@shivan-s in https://github.com/LangTrans/LangTrans/issues/7#issuecomment-905930680_
@akshaya-hub in https://github.com/LangTrans/LangTrans/issues/7#issuecomment-1420333633
Continue such discussions here

**Reply by shivan-s** [reply]
Thank you :)

**Comment by thatvideoshopguy** [comment]
This could be related to the discussion here #24
